contracts/test/MapleGlobals.t.sol

```diff
@@ -1,273 +1,439 @@
+        (Governor realGov, Governor fakeGov, MapleGlobals globals) = makeGlobalsAndGovs();
+
+        assertTrue(!fakeGov.try_mapleGlobals_setDefaultGracePeriod(address(globals), 1 days));
+        assertTrue( realGov.try_mapleGlobals_setDefaultGracePeriod(address(globals), 1 days));
+        assertEq(globals.defaultGracePeriod(), 1 days);
+    }
+    
+    function test_setMinLoanEquity() public {
+        (Governor realGov, Governor fakeGov, MapleGlobals globals) = makeGlobalsAndGovs();
+
+        assertTrue(!fakeGov.try_mapleGlobals_setMinLoanEquity(address(globals), 1000));
+        assertTrue( realGov.try_mapleGlobals_setMinLoanEquity(address(globals), 1000));
+        assertEq(globals.minLoanEquity(), 1000);
+        
+    }
+    
+    function test_setFundingPeriod() public {
+        (Governor realGov, Governor fakeGov, MapleGlobals globals) = makeGlobalsAndGovs();
+
+        assertTrue(!fakeGov.try_mapleGlobals_setFundingPeriod(address(globals), 1 days));
+        assertTrue( realGov.try_mapleGlobals_setFundingPeriod(address(globals), 1 days));
+        assertEq(globals.fundingPeriod(), 1 days);
+    }
+    
+    function test_setSwapOutRequired() public {
+        (Governor realGov, Governor fakeGov, MapleGlobals globals) = makeGlobalsAndGovs();
+
+        assertTrue(!fakeGov.try_mapleGlobals_setSwapOutRequired(address(globals), 10_000));
+        assertTrue( realGov.try_mapleGlobals_setSwapOutRequired(address(globals), 10_000));
+        assertEq(globals.swapOutRequired(), 10_000);
+    }
+    
+    function test_setPriceOracle() public {
+        (Governor realGov, Governor fakeGov, MapleGlobals globals) = makeGlobalsAndGovs();
+
+        assertEq(globals.oracleFor(address(1)), address(0));
+        
+        assertTrue(!fakeGov.try_mapleGlobals_setPriceOracle(address(globals), address(1), address(2)));
+        assertTrue( realGov.try_mapleGlobals_setPriceOracle(address(globals), address(1), address(2)));
+
+        assertEq(globals.oracleFor(address(1)), address(2));
+    }
+    
+    function test_setPendingGovernor() public {
+        (Governor realGov, Governor fakeGov, MapleGlobals globals) = makeGlobalsAndGovs();
+
+        assertEq(globals.pendingGovernor(), address(0));
+        
+        assertTrue(!fakeGov.try_mapleGlobals_setPendingGovernor(address(globals), address(1)));
+        assertTrue( realGov.try_mapleGlobals_setPendingGovernor(address(globals), address(1)));
+
+        assertEq(globals.pendingGovernor(), address(1));
+    }
+
+    /***************/
+    /*** Getters ***/
+    /***************/
+    function test_getLatestPrice() public {
+        Governor     realGov = new Governor();
+        MapleGlobals globals = new MapleGlobals(address(realGov), address(1), address(2));
+
+        address oracleMock = address(new OracleMock());
+        realGov.mapleGlobals_setPriceOracle(address(globals), address(1), oracleMock);
+        assertEq(globals.getLatestPrice(address(1)), 100);
+    }
```

Comment:
`makeGlobalsAndGovs` not used again. Perhaps throw this in the other contract and `MapleGlobalsGeneralTests`.